packages/office-ui-fabric-react/src/components/pickers/examples/TagPicker.Basic.Example.tsx

```diff
@@ -24,8 +23,13 @@ const _testTags = [
   'yellow'
 ].map(item => ({ key: item, name: item }));
 
+export interface ITagPickerDemoPageState {
```

Comment:
>export [](start = 0, length = 6)

nit: unnecessary export

Reply:
Resolved. I moved it from a separate file and forgot to remove the export)))